### ISSUE TITLE
Process killed shortly after announcing "Moving model to GPU ..." in the UI

### ISSUE THREAD

**Issue by Mister-Rub** [lllyasviel/Fooocus#1116]
**What is the problem**
After submitting a prompt, the generation starts and then stop when the process is killed, the UI announcing at crash time "Moving model to GPU ...". 
I am on Ubuntu 22.04.3 LTS ,  RTX 3050 Ti Mobile 4GiB VRM and 16 GiB, installed via venv. 

**Full Console Log**
Already up-to-date
Update succeeded.
[System ARGV] ['entry_with_update.py']
Python 3.10.12 (main, Nov 20 2023, 15:14:05) [GCC 11.4.0]
Fooocus version: 2.1.824
Running on local URL:  http://127.0.0.1:7865

To create a public link, set `share=True` in `launch()`.
Total VRAM 3905 MB, total RAM 15727 MB
Trying to enable lowvram mode because your GPU seems to have 4GB or less. If you don't want this use: --normalvram
Set vram state to: LOW_VRAM
Disabling smart memory management
Device: cuda:0 NVIDIA GeForce RTX 3050 Ti Laptop GPU : native
VAE dtype: torch.bfloat16
Using pytorch cross attention
Refiner unloaded.
model_type EPS
adm 2816
Using pytorch attention in VAE
Working with z of shape (1, 4, 32, 32) = 4096 dimensions.
Using pytorch attention in VAE
extra keys {'cond_stage_model.clip_l.logit_scale', 'cond_stage_model.clip_g.transformer.text_model.embeddings.position_ids', 'cond_stage_model.clip_l.text_projection'}
Base model loaded: /home/username/AI/Fooocus/models/checkpoints/juggernautXL_version6Rundiffusion.safetensors
Request to load LoRAs [['sd_xl_offset_example-lora_1.0.safetensors', 0.1], ['None', 1.0], ['None', 1.0], ['None', 1.0], ['None', 1.0]] for model [/home/username/AI/Fooocus/models/checkpoints/juggernautXL_version6Rundiffusion.safetensors].
Loaded LoRA [/home/username/AI/Fooocus/models/loras/sd_xl_offset_example-lora_1.0.safetensors] for UNet [/home/username/AI/Fooocus/models/checkpoints/juggernautXL_version6Rundiffusion.safetensors] with 788 keys at weight 0.1.
Fooocus V2 Expansion: Vocab with 642 words.
Fooocus Expansion engine loaded for cpu, use_fp16 = False.
Requested to load SDXLClipModel
Requested to load GPT2LMHeadModel
Loading 2 new models
App started successful. Use the app with http://127.0.0.1:7865/ or 127.0.0.1:7865
[Parameters] Adaptive CFG = 7
[Parameters] Sharpness = 2
[Parameters] ADM Scale = 1.5 : 0.8 : 0.3
[Parameters] CFG = 4.0
[Parameters] Seed = 8381210765946020217
[Parameters] Sampler = dpmpp_2m_sde_gpu - karras
[Parameters] Steps = 30 - 15
[Fooocus] Initializing ...
[Fooocus] Loading models ...
Refiner unloaded.
[Fooocus] Processing prompts ...
[Fooocus] Preparing Fooocus text #1 ...
[Prompt Expansion] Chef baking cookies, dramatic atmosphere, detailed, intricate, elegant, highly dynamic light, sharp focus, professional composition, colorful, beautiful, surreal, new classic, cool, creative, perfect color, ambient, cinematic, symmetry, great quality, pure, fine detail, extremely aesthetic, glowing, innocent, cute, magical, artistic, balanced, calm, lovely, complex
[Fooocus] Preparing Fooocus text #2 ...
[Prompt Expansion] Chef baking cookies, bright colors, elegant, highly detailed, intricate, sharp focus, beautiful light, epic composition, joyful, colorful background, cute, expressive, symmetry, warm color, striking, rich deep complex artistic, romantic, sublime, cinematic, enhanced, loving, thought, imposing, amazing, attractive, cool, lovely, pretty, great, perfect, focused
[Fooocus] Encoding positive #1 ...
[Fooocus] Encoding positive #2 ...
[Fooocus] Encoding negative #1 ...
[Fooocus] Encoding negative #2 ...
[Parameters] Denoising Strength = 1.0
[Parameters] Initial Latent shape: Image Space (896, 1152)
Preparation time: 5.80 seconds
[Sampler] refiner_swap_method = joint
[Sampler] sigma_min = 0.0291671771556139, sigma_max = 14.614643096923828
Requested to load SDXL
Loading 1 new model
Killed

**Comment by lorenzo-frittoli** [comment]
I have the same issue. I belive the process is killed because it uses too much ram. I have 16 GB and the application filled them up (as well as the swap) right before being terminated.

**Comment by mikey0223** [comment]
Just increase your swapfile size or make a swap file. I followed this site to do so https://itsfoss.com/create-swap-file-linux/

**Comment by Mister-Rub** [comment]
Thanks, indeed I had no swap memory and for some reason without any task Fooocus already uses 70% of my memory. I added 16GiB of swapfile and **Fooocus works now**, using 7 GiB of swap memory in addition to 16GiB of RAM.

**Comment by RyelandSchool** [comment]
Thanks, this worked for me as well! I am on Ubuntu 22.03.4 with an NVIDIA 3060 GPU and 16GB of ram. Adding 16Gb of swap space fixed my issue!

**Comment by bokorumed** [comment]
I face the same issue on Windows 10 Pro, RTX 1650i 4gbVram and 8ddr4 3200, does anyone else have this issue?
